sycl/include/sycl/ext/oneapi/experimental/device_architecture.hpp

```diff
@@ -439,167 +511,247 @@ static constexpr bool is_allowable_aot_mode =
+  return ext::oneapi::experimental::architecture::nvidia_gpu_sm_60;
+#endif
+#if __SYCL_TARGET_NVIDIA_GPU_SM61__
+  return ext::oneapi::experimental::architecture::nvidia_gpu_sm_61;
+#endif
+#if __SYCL_TARGET_NVIDIA_GPU_SM62__
+  return ext::oneapi::experimental::architecture::nvidia_gpu_sm_62;
+#endif
+#if __SYCL_TARGET_NVIDIA_GPU_SM70__
+  return ext::oneapi::experimental::architecture::nvidia_gpu_sm_70;
+#endif
+#if __SYCL_TARGET_NVIDIA_GPU_SM72__
+  return ext::oneapi::experimental::architecture::nvidia_gpu_sm_72;
+#endif
+#if __SYCL_TARGET_NVIDIA_GPU_SM75__
+  return ext::oneapi::experimental::architecture::nvidia_gpu_sm_75;
+#endif
+#if __SYCL_TARGET_NVIDIA_GPU_SM80__
+  return ext::oneapi::experimental::architecture::nvidia_gpu_sm_80;
+#endif
+#if __SYCL_TARGET_NVIDIA_GPU_SM86__
+  return ext::oneapi::experimental::architecture::nvidia_gpu_sm_86;
+#endif
+#if __SYCL_TARGET_NVIDIA_GPU_SM87__
+  return ext::oneapi::experimental::architecture::nvidia_gpu_sm_87;
+#endif
+#if __SYCL_TARGET_NVIDIA_GPU_SM89__
+  return ext::oneapi::experimental::architecture::nvidia_gpu_sm_89;
+#endif
+#if __SYCL_TARGET_NVIDIA_GPU_SM90__
+  return ext::oneapi::experimental::architecture::nvidia_gpu_sm_90;
+#endif
+#if __SYCL_TARGET_AMD_GPU_GFX700__
+  return ext::oneapi::experimental::architecture::amd_gpu_gfx700;
+#endif
+#if __SYCL_TARGET_AMD_GPU_GFX701__
+  return ext::oneapi::experimental::architecture::amd_gpu_gfx701;
+#endif
+#if __SYCL_TARGET_AMD_GPU_GFX702__
+  return ext::oneapi::experimental::architecture::amd_gpu_gfx702;
+#endif
+#if __SYCL_TARGET_AMD_GPU_GFX801__
+  return ext::oneapi::experimental::architecture::amd_gpu_gfx801;
+#endif
+#if __SYCL_TARGET_AMD_GPU_GFX802__
+  return ext::oneapi::experimental::architecture::amd_gpu_gfx802;
+#endif
+#if __SYCL_TARGET_AMD_GPU_GFX803__
+  return ext::oneapi::experimental::architecture::amd_gpu_gfx803;
+#endif
+#if __SYCL_TARGET_AMD_GPU_GFX805__
+  return ext::oneapi::experimental::architecture::amd_gpu_gfx805;
+#endif
+#if __SYCL_TARGET_AMD_GPU_GFX810__
+  return ext::oneapi::experimental::architecture::amd_gpu_gfx810;
+#endif
+#if __SYCL_TARGET_AMD_GPU_GFX900__
+  return ext::oneapi::experimental::architecture::amd_gpu_gfx900;
+#endif
+#if __SYCL_TARGET_AMD_GPU_GFX902__
+  return ext::oneapi::experimental::architecture::amd_gpu_gfx902;
+#endif
+#if __SYCL_TARGET_AMD_GPU_GFX904__
+  return ext::oneapi::experimental::architecture::amd_gpu_gfx904;
+#endif
+#if __SYCL_TARGET_AMD_GPU_GFX906__
+  return ext::oneapi::experimental::architecture::amd_gpu_gfx906;
+#endif
+#if __SYCL_TARGET_AMD_GPU_GFX908__
+  return ext::oneapi::experimental::architecture::amd_gpu_gfx908;
+#endif
+#if __SYCL_TARGET_AMD_GPU_GFX909__
+  return ext::oneapi::experimental::architecture::amd_gpu_gfx909;
+#endif
+#if __SYCL_TARGET_AMD_GPU_GFX90a__
+  return ext::oneapi::experimental::architecture::amd_gpu_gfx90a;
+#endif
+#if __SYCL_TARGET_AMD_GPU_GFX90c__
+  return ext::oneapi::experimental::architecture::amd_gpu_gfx90c;
+#endif
```

Comment:
@dm-vodopyanov shouldn't these have been `__SYCL_TARGET_AMD_GPU_GFX90A__` and `__SYCL_TARGET_AMD_GPU_GFX90C__` (note the uppercase `A` and `C`) to match the definitions above ?